.vortex/docs/content/contributing/maintenance/release_template.md

```
@@ -0,0 +1,136 @@
+- [ ] Updated container images to the latest versions and checked that `@see` links
+- [ ] Updated PHP version in `composer.json` for `config.platform`.
+- [ ] Updated PHP version in `phpcs.xml` for `testVersion`.
+- [ ] Updated PHP version in `phpstan.neon` for `phpVersion`.
+- [ ] Updated minor version of all packages in `composer.json`.
+- [ ] Updated minor version of dependencies in theme's `package.json`.
+- [ ] Incremented the cache version in `.circleci/config.yml` and `.github/workflows/build-test-deploy.yml`.
+- [ ] Updated documentation.
+
+---
+
+**Full Changelog**: https://github.com/drevops/vortex/compare/[PREVIOUS_VERSION]...[NEW_VERSION]
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)